docs/en_US/Compressor/Benchmark.md

```diff
@@ -0,0 +1,88 @@
+To provide an initial insight into the performance of various channel pruning algorithms, 
+we conduct extensive experiments with various pruning algorithms on some benchmark models and datasets.
+We present the experiment result in this document.
+In addition, we provide friendly instructions on the re-implementation of these experiments to facilitate further contributions to this effort.
+
+## Experiment Setting
+
+The experiments are performed with the following pruners/datasets/models:
+
+* Models: [VGG16, ResNet18, ResNet50](https://github.com/microsoft/nni/tree/master/examples/model_compress/models/cifar10)
+
+* Datasets: CIFAR-10
+
+* Pruners: 
+    - These pruners are included:
+        - Pruners with scheduling : SimulatedAnnealing, NetAdapt, AutoCompress
+        - One-shot pruners: L1Filter, L2Filter, FPGMPruner
+    - Only **channel pruning** performances are compared here. 
+
+    For the auto-pruners, `L1FilterPruner` is used as the base algorithm. That is to say, after the sparsities distribution among the layers is decided by the scheduling algorithm, `L1FilterPruner` is used to performn real pruning.
+
+    - All the pruners listed above are implemented in [nni](https://github.com/microsoft/nni/tree/master/docs/en_US/Compressor/Overview.md).
+
+## Experiment Result
+
+For each dataset/model/pruner combination, we prune the model to different levels by setting a series of target sparsities for the pruner. 
+
+Here we plot both **Number of Weights - Performances** curve and **FLOPs - Performance** curve. 
+As a reference, we also plot the result declared in the paper [AutoCompress: An Automatic DNN Structured Pruning Framework for Ultra-High Compression Rates](http://arxiv.org/abs/1907.03141) for models VGG16 and ResNet18 on CIFAR-10.
+
+The experiment result are shown in the following figures:
+
+CIFAR-10, VGG16:
+
+![](../../../examples/model_compress/experiment_result/img/performance_comparison_vgg16.png)
+
+CIFAR-10, ResNet18:
+
+![](../../../examples/model_compress/experiment_result/img/performance_comparison_resnet18.png)
+
+CIFAR-10, ResNet50:
+
+![](../../../examples/model_compress/experiment_result/img/performance_comparison_resnet50.png)
+
+## Analysis
+
+From the experiment result, we get the following conclusions:
+
+* Given the constraint on the number of parameters, the pruners with scheduling ( `AutoCompress` , `SimualatedAnnealing` ) performs better than the others when the constraint is strict. However, they have no such advantage in FLOPs/Performances comparison since only number of parameters constraint is considered in the optimization process; 
+* The basic algorithms `L1FilterPruner` , `L2FilterPruner` , `FPGMPruner` performs very similarly in these experiments; 
```

Comment:
good summary. for this one, can I say for some model/dataset one-shot pruners perform similar to the pruners with scheduling, even though they set the same sparsity for every layer?

Reply:
> good summary. for this one, can I say for some model/dataset one-shot pruners perform similar to the pruners with scheduling, even though they set the same sparsity for every layer?

We cannot say pruners with scheduling and basic pruners are `similar` generally because they are different given different evaluation metrics.

Reply:
Isn't FLOPs related to just filter number and channel number(which are actually the same thing)? And filter number also represents parameters. How does it not improve FLOPs? I can imagine some architecture like xception won't improve because filter number reduction may not affect channel number. But it doesn't make sense for VGG16. Is it the same situation? Please correct me if I am wrong 😄

Reply:
> Isn't FLOPs related to just filter number and channel number(which are actually the same thing)? And filter number also represents parameters. How does it not improve FLOPs? I can imagine some architecture like xception won't improve because filter number reduction may not affect channel number. But it doesn't make sense for VGG16. Is it the same situation? Please correct me if I am wrong 😄

Thanks for your reply. FLOPs is also related to image resolution, the input size of different layers are different.

Reply:
Oh thanks for reminding me. Aren't Input size/feature map size almost the same size no matter which pruner? If so, the FLOPs should be correspond to params.